docs(cpt): requirements section revamp (#2842)
scaleway/docs-content: observability/cockpit/api-cli/configuring-grafana-agent.mdx

This page explains how to configure the Grafana agent and the Zipkin collector to push your metrics, logs and traces and visualize them in Grafana.

## Requirements

> - Be the [Owner](/identity-and-access-management/iam/concepts/#owner) of the Organization in which the actions will be performed, or an IAM user with the [necessary permissions](/identity-and-access-management/iam/concepts/#permission)

Author: ```suggestion
- [Owner](/identity-and-access-management/iam/concepts/#owner) status or [IAM permissions](/identity-and-access-management/iam/concepts/#permission) allowing you to perform actions in the intended Organization
```

- A Scaleway account logged into the [console](https://console.scaleway.com)
- An active [Cockpit](/observability/cockpit/how-to/activate-cockpit/)

> - [Docker](https://www.docker.com/) and [Docker Compose](https://docs.docker.com/compose/install/)

Reviewer: ```suggestion
- Installed [Docker](https://www.docker.com/) and [Docker Compose](https://docs.docker.com/compose/install/)
```
